test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
there is coverage metics, however coverage of external lib support does not exist. It is more of a boolean does it work y/n kind of thing currently